app/src/main/res/layout/activity_pcp_information_page.xml

```diff
@@ -72,31 +73,38 @@
         app:layout_constraintHorizontal_bias="0.503"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
+        android:layout_marginTop="8dp"
         app:layout_constraintTop_toBottomOf="@+id/pcp_information_email_edit_text" />
 
     <Button
         android:id="@+id/pcp_information_back_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="80dp"
-        android:layout_marginLeft="85dp"
+        android:layout_marginBottom="51dp"
+        android:layout_marginLeft="25dp"
         android:text="Back"
+        android:textSize="14sp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent" />
 
     <Button
         android:id="@+id/pcp_information_next_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="80dp"
-        android:layout_marginRight="85dp"
+        android:layout_marginBottom="51dp"
+        android:layout_marginRight="16dp"
         android:text="Next"
+        android:textSize="14sp"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        android:layout_marginTop="8dp"
-        android:layout_marginLeft="8dp"
-        app:layout_constraintLeft_toLeftOf="parent" />
+        app:layout_constraintRight_toRightOf="parent" />
+
+    <QuickContactBadge
```

Comment:
are we going to do something with this? 

can you look into how we insert a picture, please?